src/object/fluid.rs

```diff
@@ -165,13 +165,6 @@ impl Fluid {
         self.positions.len()
     }
 
-    /// Computes the AABB of this fluid.
-    #[cfg(feature = "nphysics")]
-    pub fn compute_aabb(&self, particle_radius: Real) -> ncollide::bounding_volume::AABB<Real> {
-        use ncollide::bounding_volume::{self, BoundingVolume};
-        bounding_volume::local_point_cloud_aabb(&self.positions).loosened(particle_radius)
-    }
-
```

Comment:
This should probably be kept but translated for parry ?

Reply:
Agreed. The translation should look very similar. Parry’s point-cloud calculation function is in `bounding_volume::details::local_point_cloud_aabb`.